src/ui.js

```diff
@@ -0,0 +1,79 @@
+const display = () => {
+  const listData = JSON.parse(localStorage.getItem('tasks')) || [];
+  const list = document.querySelector('.container2');
+  list.innerHTML = '';
+  if (listData.length > 0) {
+    listData.forEach((task) => {
+      const { description, index } = task;
+      list.innerHTML += `
+      <li class="task-container">
+      <div class="task">
+      <div class="task-text">
+      <input type="checkbox" id="${index}" class="check-box" name="task"/><del class="input-text" contenteditable="false">${description}</del>
+      </div>
+      <div class="edit-icon "></div>
+      <i class="trash trash-btn fa fa-trash" aria-hidden="true"></i>
+      </div>
+      <hr class="list-line" />
+      </li>
+      `;
+      const tasks = document.querySelectorAll('.task-container');
+      const inputTexts = document.querySelectorAll('.input-text');
+      const editBtns = document.querySelectorAll('.edit-icon');
+      let trashBtns = document.querySelectorAll('.trash-btn');
+      const checkBoxes = document.querySelectorAll('input[type=checkbox]');
+
+      editBtns.forEach((editBtn, index) => {
+        editBtn.addEventListener('click', () => {
+          editBtn.classList.add('hide');
+          tasks[index].classList.add('edit-mode');
+          trashBtns[index].classList.remove('trash');
+          inputTexts[index].contentEditable = 'true';
+          inputTexts[index].setAttribute('contenteditable', 'true');
+          localStorage.setItem('tasks', JSON.stringify(listData));
+        });
+      });
+
+      inputTexts.forEach((inputText, index) => {
+        inputText.addEventListener(
+          'input',
+          () => {
+            listData[index].description = inputText.textContent;
+            localStorage.setItem('tasks', JSON.stringify(listData));
+          },
+          false,
+        );
+      });
+
+      checkBoxes.forEach((checkBox, index) => {
+        if (listData[index].completed === true) {
+          checkBox.checked = true;
+          inputTexts[index].previousElementSibling.disabled = true;
+        }
+
+        checkBox.addEventListener('change', () => {
+          listData[index].completed = true;
+          inputTexts[index].disabled = true;
+          inputTexts[index].previousElementSibling.disabled = true;
+          localStorage.setItem('tasks', JSON.stringify(listData));
+        });
+      });
```

Comment:
- [ ] Wonderful job for putting your project this far. I must give you accolade for the awesome job you've done. There's a little thing you need to work on before your project can be approved. It is required that a user must be able to mark a task as completed and remove the mark too. The checkbox shouldn't be disabled after a task has been marked as completed. At this moment, I can only mark tasks as completed, but not other way round. Kindly look into that and fix.
[app-behav.webm](https://user-images.githubusercontent.com/26597432/215286956-d6fa8c13-ca85-4f8f-b96d-c6fd913bc9f7.webm)


![prog-rqts](https://user-images.githubusercontent.com/26597432/215286728-12cb113e-ff95-4c52-8ac9-36bb3221144f.png)


- [ ] It is also a requirement to create a separate file and import it as a module. That file should contain methods related to the status updates (completed: true / false). Kindly create a new file, move these methods in there and import it as a module wherever you plan to use them.
![project-req](https://user-images.githubusercontent.com/26597432/215286739-646efe84-06bf-4bd5-92c1-d2d5e01eace5.png)